<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serach</class>
 <widget class="QWidget" name="Serach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LabelUserName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBoxSearchKind">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSearchKind">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询方式</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditSearchContent">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelSearchContent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入查询内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonClearnText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空查询框</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonApplyFor">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>申请证书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonToSearch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCertID">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>证书编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCert">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>证书：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditCertID">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditCert">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCertState">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>证书状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelClientName">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>申请者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditClientName">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditCertState">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>450</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDelete">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>550</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除此证书</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCertCreateTime">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditCertCreateTime">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCertDieTime">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>到期时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEditCertDieTime">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
